### PR TITLE
Requests

### PR DESCRIPTION
**Pull request by jannelie**
As a developer I want to bot to be able to respond to requests in the game with a certain trategy so that it can make a nice game.

Testing is done manually, see test document on how to repeat this test.

### CODE REVIEW

**Comment by ardwennem2** [comment]
Ziet er goed uit volgens mij

**Comment by HarmenKroon** [comment]
Als ik het goed begrijp komen nu alle requests in de belief base; ongeacht of ze voor ons zijn of iemand anders.
Kan dit verholpen worden?

**Comment by jannelie** [comment]
REquests are now only added if they include our stakeholder.

File: src/main/goal/nl/tudelft/ti2806/request.mod2g [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	%				You can only have an OK option if there are no other options.
+	% if:
+	% - you have an interaction request
+	% - this request is to sell some land
+	% - it is possible to answer with Yes or OK 
+	% then:
+	% - answer the reqest with Yes or OK.
+	if
+		bel(
+		my_stakeholder_id(StakeholderID),
```
Comment (nskartoredjo) [inline src/main/goal/nl/tudelft/ti2806/request.mod2g]: should have an aditional tab (there are more lines like these)

File: src/main/goal/nl/tudelft/ti2806/service.act2g [hunk 2]
```diff
@@ -36,3 +36,13 @@ define	map_buy_land(MultiPolygon, Price) with
 define	map_sell_land(BuyerId,MultiPolygon, Price) with
 	pre{ true }
 	post{ true}
+
+% description:
+% - answer to a certain popup.
+% parameters:
+% -	PopUpID: The popup we are answering
```
Comment (nskartoredjo) [inline src/main/goal/nl/tudelft/ti2806/service.act2g]: unnecessary tab

File: src/main/goal/nl/tudelft/ti2806/service.mod2g [hunk 3]
```diff
@@ -16,6 +17,24 @@ order = linear.
...
+		Type =\= 'PERMIT'
+	)
+	then
+		request.
+	
+	
+	
+	
+	
+
```
Comment (nskartoredjo) [inline src/main/goal/nl/tudelft/ti2806/service.mod2g]: unnecessary white space

**Comment by nskartoredjo** [comment]
every directory should be lower case, including 'Testing'

File: src/main/goal/nl/tudelft/ti2806/service.act2g (hunk 2 above)
Comment (jannelie) [inline src/main/goal/nl/tudelft/ti2806/service.act2g]: in my code, it doesn't have this tab.

**Comment by jannelie** [comment]
refactored comments and changed folder name

**Comment by nskartoredjo** [comment]
I approve

**Comment by nskartoredjo** [comment]
Approved

**Comment by jannelie** [comment]
nice to add pullapprove, I approve this addition ;)

**Comment by jannelie** [comment]
Just to test if I can approve to..
Approved

**Comment by ardwennem2** [comment]
Approved

**Comment by ardwennem2** [comment]
Approved

**Comment by jannelie** [comment]
solved conflicts

**Comment by nskartoredjo** [comment]
Merge conflicts are resolved and the Travis build succeeded

Approved

**Pull request merged**